Apps/WebAPI/sdLitica.WebAPI/Models/Security/CustomAuthHandler.cs

```diff
@@ -53,13 +53,26 @@ protected override async Task<AuthenticateResult> HandleAuthenticateAsync()
             Guid userId = default(Guid);
             try
             {
+                string schema = ((string)authorization)?.Split(' ', 2)[0]; 
                 string token = ((string)authorization)?.Split(' ', 2)[1];
-                UserToken userToken = await userService.GetByTokenAsync(token);
-                if (userToken == null || userToken.IsTokenExpired())
-                    throw new UnauthorizedAccessException();
 
-                await userService.ShiftTokenAsync(userToken);
-                userId = userToken.UserId;
+                if (schema.ToLower().Equals("cloudtoken"))
+                {
+                    UserToken userToken = await userService.GetByTokenAsync(token);
+                    if (userToken == null || userToken.IsTokenExpired())
+                        throw new UnauthorizedAccessException();
+
+                    await userService.ShiftTokenAsync(userToken);
+                    userId = userToken.UserId;
+                } 
+                else if (schema.ToLower().Equals("cloudapikey"))
```

Comment:
- [ ] Maybe we should move `"cloudtoken"` and `"cloudapikey"` constants to `CustomAuthOptions` class or some new enum like a `CustomAuthSchemas`, and use it from `AuthenticationHandler<T>.Options` property like in [#L35](https://github.com/sdLiticaProject/code/blob/issue/44-Implement-an-ability-to-end-user-to-create-an-API-key-and-authenticate-requests-with-API-Key/Apps/WebAPI/sdLitica.WebAPI/Models/Security/CustomAuthHandler.cs#L35) ? 
- [ ] Is there a need to add a new `"cloudapikey"` scheme in [`AuthenticationBuilderExtensions#L13`](https://github.com/sdLiticaProject/code/blob/076b6452aaccbdb6338010a9b52afbd5cef18328/Apps/WebAPI/sdLitica.WebAPI/Models/Security/AuthenticationBuilderExtensions.cs#L13) ?
- [ ] What happens if `schema` variable is neither `"cloudtoken"` nor `"cloudapikey"`? It looks like we don't fall in any `if/else if` branches and don't throw any exceptions, but successfully return `Claim` with _default(Guid).ToString()_
- [ ] Even if `schema` equals `"cloudapikey"`, we still return `identities` and `tiket` with `Options.Scheme` value which equals to `"cloudToken"`. See [`CustomAuthOptions.cs#L9-L10`](https://github.com/sdLiticaProject/code/blob/076b6452aaccbdb6338010a9b52afbd5cef18328/Apps/WebAPI/sdLitica.WebAPI/Models/Security/CustomAuthOptions.cs#L9-L10)